obal/data/repoclosure.yml

```diff
@@ -0,0 +1,32 @@
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: 'Build additional repos'
+      set_fact:
+        additional_repos: "{{ additional_repos|default([]) + ['--repofrompath=' + item.name + ',' + item.url] }}"
+      with_items: "{{ repoclosure_additional_repos }}"
+      when: repoclosure_additional_repos is defined
+
+    - debug:
+        msg: "{{ additional_repos }}"
+
+    - name: 'Run repoclosure'
+      command: >
+        repoclosure
+          -c {{ inventory_dir }}/{{ repoclosure_config }}
+          -t
+          -r {{ repoclosure_target_repo }}
+          {{ additional_repos | join(' ') }}
+          {{ '-l ' if repoclosure_look_aside_repos else '' }}
+          {{ repoclosure_look_aside_repos | join(' -l ') }}
+      register: output
+      when: output is not defined
```

Comment:
Why is this `when` necessary?

Reply:
The ansible-lint rule not to run operations if there are no changes.